### PR TITLE
BFW-698 v.2

### PR DESCRIPTION
**Pull request by Chleba**
Zruseni pridane menu screeny "Tune Info" a premisteni jejich polozek primo do "Tune Menu".

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -15,5 +15,6 @@
     ],
     "cmake.buildDirectory": "${workspaceFolder}/build-vscode",
     "cmake.cmakePath": "${workspaceFolder}/.dependencies/cmake-3.15.5/bin/cmake",
-    "cmake.generator": "Ninja"
+    "cmake.generator": "Ninja",
```
Comment (DRracer) [inline .vscode/settings.json:18]: please don't include settings.json and similar preset configuration files in PR's

File: src/gui/guimain.c [hunk 2]
```diff
@@ -246,6 +246,7 @@ void gui_run(void) {
         gui_loop();
 #ifndef LCDSIM
         if (marlin_message_received()) {
+
```
Comment (DRracer) [inline src/gui/guimain.c:249]: new line - an unnecessary change, please remove it from the PR

File: src/gui/screen_menu_tune.cpp [hunk 3]
```diff
@@ -161,9 +164,11 @@ int screen_menu_tune_event(screen_t *screen, window_t *window,
         case MI_BABYSTEP:
             z_offs = psmd->items[MI_BABYSTEP].item.wi_spin_fl.value;
             break;
-        case MI_MESSAGES:
+            // je potreba mit ? - stara se o to menu_screen_event
```
Comment (DRracer) [inline src/gui/screen_menu_tune.cpp:167]: please keep comments in EN
And processing MI_MESSAGES is not necessary in this menu

File: src/guiapi/src/screen.c [hunk 4]
```diff
@@ -3,9 +3,9 @@
 #include "screen.h"
 #include "gui.h"
 
-#define SCREEN_MAX_SCREENS 32
+#define SCREEN_MAX_SCREENS 33
```
Comment (DRracer) [inline src/guiapi/src/screen.c:6]: increasing MAX_SCREENS is not necessary in this PR

Comment (Chleba) [reply src/guiapi/src/screen.c]: opraveno zpet na 32
